eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+# eMPRess_GUI_iteration5
+
+import tkinter as tk
+from tkinter import messagebox
+from pathlib import Path
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib.backend_bases import MouseEvent, key_press_handler
+import empress
+
+class App:
+
+    def __init__(self, master):
+        self.master = master
+        ### configures the master frame ###
+        master.grid_rowconfigure(0, weight=1)
+        master.grid_rowconfigure(1, weight=2)
+        master.grid_columnconfigure(0, weight=1)
+        master.grid_columnconfigure(1, weight=1)
+
+        ### creates a logo frame on top of the master frame ###
+        self.logo_frame = tk.Frame(master)
+        self.logo_frame.grid(row=0, column=0, columnspan=2, sticky="nsew")
+        self.logo_frame.grid_propagate(False)
+
+        # adds background image
+        photo = tk.PhotoImage(file="JaneTransLogo-thin.png")
+        label = tk.Label(self.logo_frame, image=photo)
+        label.place(x=0, y=0)
+        label.image = photo
+
+        ### creates an input frame on the left side of the master frame ###
+        self.func_frame = tk.Frame(master)
+        self.func_frame.grid(row=1, column=0, sticky="nsew")
+        self.func_frame.grid_rowconfigure(0, weight=1)
+        self.func_frame.grid_rowconfigure(1, weight=1)
+        self.func_frame.grid_rowconfigure(2, weight=1)
+        self.func_frame.grid_rowconfigure(3, weight=1)
+        self.func_frame.grid_columnconfigure(0, weight=1)
+        self.func_frame.grid_propagate(False)
+
+        ### creates an output frame on the right side of the master frame ###
+        self.output_frame = tk.Frame(master)
+        self.output_frame.grid(row=1, column=1, sticky="nsew")
```

Comment:
What does sticky mean? If the meaning is not obvious, it might be a good idea to add a comment.

Reply:
It means that self.output_frame expands in all four directions (north, south, east and west) to fully occupy the allocated space in the grid system (row 1 column 1). I will add the comment. :)

Reply:
The comment was added for the logo_frame when sticky first appeared. :)